portal/templates/instance_profile.html

```diff
@@ -114,5 +134,27 @@ <h1>Detailed services information currently not available</h1>
     }
 }
 
+var instance = $('#instance-data').data();
+var instance_id = instance.name;
+
+var request_logs = new Vue({
+  el: '#logs-display',
+  data: {
+    logs: null
+  },
+  methods: {
+    getLogs: function () {
+      var lines = document.getElementById('nlines').value
+      var container = document.getElementById('container-list').value
+      axios.get('/instances-log-xhr/' + instance_id + '/' + container + '/' + lines)
+      .then(response => this.logs = response.data)
+      .catch(error => this.logs = "Logs currently unavailable")
```

Comment:
`.catch` returns "Logs currently unavailable" message. this replaces the original `if instance_log['logs']` conditional statement